uber/templates/macros.html

```diff
@@ -117,4 +117,113 @@
       </div>
     {%- endfor %}
   </div>
-{%- endmacro %}
\ No newline at end of file
+{%- endmacro %}
+
+{% macro address_form(group=False, attendee=False, is_prereg_dealer=False, copy_address=False) -%}
+    {% set name_prefix='group_' if group and attendee else '' %}
+    {% set group_or_attendee=group if group else attendee %}
+    {% if not is_prereg_dealer %}
+        <script src="../static/deps/selectToAutocomplete/jquery.select-to-autocomplete.js"></script>
+        <script type="text/javascript">{% include "region_opts.html" %}</script>
+    {% endif %}
+    <script type="text/javascript">
```

Comment:
I have some concerns about including javascript in a macro that may be used more than once on a single page. This will mostly work, but may cause issues, and it's definitely less efficient. For example:
```javascript
$(function () {
    if ($('#copy_address').length) {
        copyAddress();
        $('#copy_address').change(copyAddress);
    }
})
```

I'm pretty sure that's going to add an additional event handler for each time the macro appears on a single page. Everything will _probably_ still work fine, but it's not ideal.